include/mbedtls/md.h

```diff
@@ -168,9 +168,9 @@ void mbedtls_md_free( mbedtls_md_context_t *ctx );
  * \param md_info   The information structure of the message-digest algorithm
  *                  to use.
  *
- * \returns         \c 0 on success,
- *                  #MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter failure,
- *                  #MBEDTLS_ERR_MD_ALLOC_FAILED memory allocation failure.
+ * \returns         \c 0 on success.
+ * \returns         #MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter failure.
```

Comment:
We're switching between `\return` and `\returns` here. This should be uniformized.